### DISCUSSION TITLE
hi there! i cant see stats, why so ?

### DISCUSSION THREAD

**Discussion by ipsrulz** [projectdiscovery/naabu#848] [Q&A]
hey bro do you have any idee how can ii see scanning stats ?  i tryied in many ways but still nothing. thank u!

**Comment by MetzinAround** [comment]
Hi @ipsrulz, could you tell us a few things:

- what version of naabu you are using
- what command you're running
- what the output is

Also, the `-stats` flag should be outputting stats, but you can also pair it with `-v` and `-debug` to see more info about scans.
